nntrainer/tensor/cl_operations/cl_sgemv.cpp

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 Debadri Samaddar <s.debadri@samsung.com>
+ *
+ * @file    cl_sgemv.cpp
+ * @date    06 Feb 2024
+ * @see     https://github.com/nnstreamer/nntrainer
+ * @author  Debadri Samaddar <s.debadri@samsung.com>
+ * @bug     No known bugs except for NYI items
+ * @brief   Experimental SGEMV implementation using OpenCL
+ *
+ * @note This file is experimental and is kept for testing purpose
+ *
+ */
+
+#include "cl_sgemv.h"
+#include <iostream>
+#include <opencl_buffer.h>
+
+#include <nntrainer_log.h>
+
+namespace nntrainer::internal {
+
+template <typename T>
+T *GpuCLSgemv::CLSgemv(const T *matAdata, const T *vecXdata, T *vecYdata,
+                       T alpha, T beta, unsigned int dim1, unsigned int dim2) {
+
+  ml_logi("GpuCLSgemv::CLSgemv");
```

Comment:
is this needed?

Reply:
This file has been kept for testing/experimental purpose as of now, hence the logs. Might be removed later.

Reply:
gotcha